scripts/toaster-layers-recipes-config.py

```diff
@@ -40,9 +40,14 @@
 
 # Now time to populate custom.xml
 
-for layername in layer_and_recipes:
+del_layers = []
+
+for layername in list(layer_and_recipes):
     if len(layer_and_recipes[layername]) == 1:
-        del layer_and_recipes[layername]
+        del_layers.append(layername)
+
+for layername in del_layers:
+    del layer_and_recipes[layername]
```

Comment:
This is pointless. Converting to list() is all you needed to do in order to make it safe to delete during the iteration. Perhaps I didn't make that clear enough in the previous review. You don't modify what you're iterating. If you use list(), then you aren't modifying what you're iterating anymore, you're iterating over a list, not the dictionary.